src/app/shell/main/main.component.html

```diff
@@ -19,9 +20,13 @@ <h3>Найпопулярніші гуртки</h3>
       </a>
     </div>
     <ng-container>
-      <div fxLayout='row wrap' fxLayoutAlign='space-between' fxLayoutGap="20px grid">
-        <app-workshop-card [parent]="parent" *ngFor="let workshop of topWorkshops$ | async " [workshop]="workshop"  [isMainPage]="true"
-          [userRole]="''">
+      <div fxLayout='row wrap' fxLayoutAlign='space-evenly' fxLayoutGap="20px grid" #WorkshopsWrap>
+        <app-workshop-card [parent]="parent" *ngFor="let workshop of topWorkshops$ | async" [workshop]="workshop"
+          [isMainPage]="true" [userRole]="''">
```

Comment:
I am wondering why we need to pass empty string as a parameter in workshop card? if we have some condition in the workshop component we should be good with undefined